### PR TITLE
Add and Refactor test code for storage

### LINKED ISSUE #92: Refactor test code for storage in quickcache.storage package

### PR DESCRIPTION
**Pull request by ChenXJ98**
closes #92

### CODE REVIEW

File: src/test/java/quickcache/storage/JsonQuickCacheStorageTest.java [hunk 1]
```diff
@@ -1,110 +1,110 @@
...
+        assertFalse(readAddressBook("NonExistentFile.json").isPresent());
+    }
+
+    @Test
+    public void read_notJsonFormat_exceptionThrown() {
+        assertThrows(DataConversionException.class, () -> readAddressBook("notJsonFormatAddressBook.json"));
+    }
+
+    @Test
+    public void readAddressBook_invalidPersonAddressBook_throwDataConversionException() {
```
Comment (FH-30) [inline src/test/java/quickcache/storage/JsonQuickCacheStorageTest.java]: There are still instances of person and addressbook here and there, might need to rename them.

**Review by FH-30 — COMMENTED** [review]
LGTM! Just need to rename some terms to fit with QuickCache and Flashcards.

**Review by GilbertTan19 — APPROVED** [review]
Looks good to merge for me! Great Job!

File: src/test/java/quickcache/storage/JsonQuickCacheStorageTest.java (hunk 1 above)
Comment (ChenXJ98) [reply src/test/java/quickcache/storage/JsonQuickCacheStorageTest.java]: I've made the changes, thanks!

**Pull request merged**